integration-cli/docker_cli_links_test.go

```diff
@@ -236,15 +200,12 @@ func (s *DockerSuite) TestLinksNetworkHostContainer(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	dockerCmd(c, "run", "-d", "--net", "host", "--name", "host_container", "busybox", "top")
 	out, _, err := dockerCmdWithError("run", "--name", "should_fail", "--link", "host_container:tester", "busybox", "true")
-	if err == nil || !strings.Contains(out, "--net=host can't be used with links. This would result in undefined behavior") {
-		c.Fatalf("Running container linking to a container with --net host should have failed: %s", out)
-	}
+	c.Assert(err, checker.NotNil, check.Commentf("Running container linking to a container with --net host should have failed: %s", out))
+	c.Assert(out, checker.Contains, "--net=host can't be used with links. This would result in undefined behavior", check.Commentf("Running container linking to a container with --net host should have failed: %s", out))
 }
 
 func (s *DockerSuite) TestLinksEtcHostsRegularFile(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	out, _ := dockerCmd(c, "run", "--net=host", "busybox", "ls", "-la", "/etc/hosts")
-	if !strings.HasPrefix(out, "-") {
-		c.Errorf("/etc/hosts should be a regular file")
-	}
+	c.Assert(strings.HasPrefix(out, "-"), check.Equals, true, check.Commentf("/etc/hosts should be a regular file"))
```

Comment:
this can be just a regular comment 
`// /etc/hosts should be a regular file`
on the same line or the previous line, and it will be printed out.